### PR TITLE
Add prisma adapter + handle query params

### PR DESCRIPTION
**Pull request by foyarash**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,140 @@
...
-### Usage
+or
 
-TODO
+`npm i -S next-crud`
 
-### Example
+# Usage
+
+- Create a file under `pages/api/<modelName>` called `[[...someName]].ts`. You can also have 2 files called `index.ts` and `[id].ts` under that folder, both files will have to contain the same content.
```
Comment (tlenclos) [inline README.md:13]: >  You can also have 2 files called `index.ts` and `[id].ts` under that folder, both files will have to contain the same content.

Why ?

Comment (tlenclos) [reply README.md:13]: I think we need to guide more on the "someName" part, what does it mean ?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,140 @@
...
+
+- Import the handler and the adapter you want to use, in that case we will use the prisma adapter but you can use your own one (see [this section](#adapters))
+
+```javascript
+import NextCrud from 'next-crud'
+import PrismaAdapter from 'next-crud/dist/adapters/prisma'
```
Comment (tlenclos) [inline README.md:19]: This should be exported in the package directly, lile 

```javascript
import NextCrud, { PrismaAdapter } from 'next-crud';
```

**Review by tlenclos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (foyarash) [reply README.md:13]: Because we can have both files notation, but if we use 2 files we still need to have the next crud handler in each one

**Pull request merged**